client/src/main/scala/ch/epfl/bluebrain/nexus/admin/client/types/Organization.scala

```diff
@@ -40,16 +40,25 @@ object Organization {
   implicit val organizationDecoder: Decoder[Organization] =
     Decoder.instance { hc =>
       for {
-        id          <- hc.get[AbsoluteIri]("@id")
-        description <- hc.get[String]("description")
-        label       <- hc.get[String]("_label")
-        uuid        <- hc.get[String]("_uuid").map(UUID.fromString)
-        rev         <- hc.get[Long]("_rev")
-        deprecated  <- hc.get[Boolean]("_deprecated")
-        createdBy   <- hc.get[AbsoluteIri]("_createdBy")
-        createdAt   <- hc.get[Instant]("_createdAt")
-        updatedBy   <- hc.get[AbsoluteIri]("_updatedBy")
-        updatedAt   <- hc.get[Instant]("_updatedAt")
-      } yield Organization(id, label, description, uuid, rev, deprecated, createdAt, createdBy, updatedAt, updatedBy)
+        id         <- hc.get[AbsoluteIri]("@id")
+        label      <- hc.get[String]("_label")
+        uuid       <- hc.get[String]("_uuid").map(UUID.fromString)
+        rev        <- hc.get[Long]("_rev")
+        deprecated <- hc.get[Boolean]("_deprecated")
+        createdBy  <- hc.get[AbsoluteIri]("_createdBy")
+        createdAt  <- hc.get[Instant]("_createdAt")
+        updatedBy  <- hc.get[AbsoluteIri]("_updatedBy")
+        updatedAt  <- hc.get[Instant]("_updatedAt")
+      } yield
+        Organization(id,
+                     label,
+                     hc.get[String]("description").toOption,
```

Comment:
You can do `description <- hc.get[Option[String]]("description")` in the for comprehension